src/lib/functions/synchronous.js

```diff
@@ -14,13 +24,9 @@ const handleSynchronousFunction = function (err, result, response) {
   }
 
   response.statusCode = result.statusCode
-  for (const key in result.headers) {
```

Comment:
We can't use `headers.forEach` directly instead of `for in`. We need to get the entries first via `Object.entries(headers)`

Reply:
Okay. Thank you.